multimedia/mlt/Portfile

```diff
@@ -80,100 +77,54 @@ depends_lib-append  port:atk \
-                    --enable-gpl \
-                    --disable-jackrack \
-                    --disable-mmx \
-                    --disable-gtk2 \
-                    --disable-swfdec \
-                    --disable-opencv \
-                    --disable-sdl \
-                    --without-kde
+                    -DGPL=ON \
+                    -DGPL3=OFF \
+                    -DMOD_JACKRACK=OFF \
+                    -DMOD_OPENCV=OFF \
+                    -DMOD_SDL1=OFF \
+                    -DMOD_QT=OFF \
+                    -DMOD_QT6=OFF
 
 if {[string match *clang* ${configure.cxx}] && ${configure.cxx_stdlib} ne ""} {
     configure.ldflags-append -stdlib=${configure.cxx_stdlib}
 }
 
-pre-configure {
-    configure.env-append \
-                    QTDIR=${qt_dir}
-    # NB: qt-includedir can and should be set to ${qt_includes_dir},
-    # not ${qt_includes_dir}/Qt !!
-    configure.args-append \
-                    --qt-libdir=${qt_frameworks_dir} \
-                    --qt-includedir=${qt_includes_dir}
-
-    configure.cflags-append  "-I${qt_includes_dir}"
-}
-
-platform darwin 10 {
-    if {${build_arch} eq "x86_64"} {
-        configure.args-delete --disable-mmx
-    }
-}
-
-configure.cflags-append     "-I${prefix}/include"
-configure.ldflags-delete    "-L${prefix}/lib"
-
-# make sure the build system actually sees our settings by amending the toplevel config.mak.
-post-configure {
-    set config.mak [open ${worksrcpath}/config.mak "a"]
-    puts ${config.mak} "CC=${configure.cc}"
-    puts ${config.mak} "CXX=${configure.cxx}"
-    puts ${config.mak} "CFLAGS+=${configure.cflags} ${configure.cppflags}"
-    puts ${config.mak} "CXXFLAGS+=${configure.cxxflags} ${configure.cppflags}"
-    puts ${config.mak} "LDLAGS+=${configure.ldflags}"
-    close ${config.mak}
-}
-
-build.args-append           CC="${configure.cc}" CXX="${configure.cxx}" CPP="${configure.cpp}"
-
-if {${universal_possible} && [variant_isset universal]} {
-    foreach arch ${configure.universal_archs} {
-        lappend merger_build_env(${arch})   "CFLAGS=${configure.cflags} -arch ${arch}"
-        lappend merger_build_env(${arch})   "CXXFLAGS=${configure.cxxflags} -arch ${arch}"
-        lappend merger_build_env(${arch})   "LDFLAGS=${configure.ldflags} -L${worksrcpath}-${arch}/src/framework -L${prefix}/lib -arch ${arch}"
-    }
-} else {
-    build.env-append                        "CFLAGS=${configure.cflags} [get_canonical_archflags cc]"
-    build.env-append                        "CXXFLAGS=${configure.cxxflags} [get_canonical_archflags cxx]"
-    build.env-append                        "LDFLAGS=${configure.ldflags} -L${worksrcpath}/src/framework -L${prefix}/lib [get_canonical_archflags ld]"
-}
-
-variant gtk2 description {include the gtk2 module (for additional image and text producers)} {
-    configure.args-delete   --disable-gtk2
-    depends_lib-append      path:lib/pkgconfig/gtk+-2.0.pc:gtk2
-    require_active_variants gtk2 x11
-}
```

Comment:
In MLT 7, the `gtk2` module was replaced with the `gdk` module to reduce dependencies and reflect its truer nature. It still has the useful "additional image and text producers" if not using Qt. Moreover, it defaults to ON. We need to either keep this variant or add the port dependencies `gdk-pixbuf2` and `pango`.

Reply:
Those dependencies seem to have been picked up via ffmpeg6 -> librsvg. I can add them manually to be explicit.